boards/arm64/intel_socfpga_agilex5_socdk/doc/index.rst

```diff
@@ -0,0 +1,14 @@
+.. _intel_socfpga_agilex5_socdk:
+
+Intel Agilex5 SoC Development Kit
+#################################
+
+Overview
```

Comment:
It would be better to put more details like other boards did?

Reply:
Will add shortly.

Reply:
Updated with all the details. Thank you.